src/components/Collective/CollectiveActions.vue

```diff
@@ -142,6 +142,7 @@ export default {
 			'isCollectiveAdmin',
 			'isPublic',
 			'loading',
+			'printLink',
```

Comment:
This seems like a leftover, `printLink()` is defined as computed property below and not available as getter from the store.